dspace/modules/api/src/main/java/org/dspace/JournalUtils.java

```diff
@@ -473,25 +477,25 @@ public static Manuscript getCrossRefManuscriptMatchingManuscript(Manuscript quer
             matchedManuscript.optionalProperties.put("crossref-score", String.valueOf(matchScore));
             // for now, scores greater than 0.5 seem to be a match. Keep an eye on this.
             if (matchScore < 0.5) {
-                log.error(queryManuscript.getTitle() + " matched " + matchedManuscript.getTitle() + " with score " + matchScore);
-                log.error("crossref url was " + crossRefURL);
+                resultString.append("\"" + queryManuscript.getTitle() + "\" matched \"" + matchedManuscript.getTitle() + "\" with score " + matchScore);
                 return null;
             }
         }
         return matchedManuscript;
     }
 
-    public static boolean compareItemAuthorsToManuscript(Item item, Manuscript manuscript) {
+    public static boolean compareItemAuthorsToManuscript(Item item, Manuscript manuscript, StringBuilder result) {
```

Comment:
little nitpick.... why is it resultString in the previous method, but the same thing is called result in this method?